### PR TITLE
Puneeth branch77

### PR DESCRIPTION
**Pull request by puneeth579**
OK

### CODE REVIEW

File: src/main/java/com/tbs/tbsm/TBSMB.java [hunk 1]
```diff
@@ -3,8 +3,19 @@
...
 	
+	public String getHow(){
+		return how;
+		
+	}
 	
+	public int getW(){
+		return w;
+	}
+
```
Comment (puneeth2020) [inline src/main/java/com/tbs/tbsm/TBSMB.java:19]: return type Integer not int.

Comment (puneeth2020) [reply src/main/java/com/tbs/tbsm/TBSMB.java:19]: Also declare it as type Integer not int

**Review by puneeth2020 — CHANGES_REQUESTED** [review]

**Comment by puneeth579** [comment]
OK... It can also be done in next release if there is no problem.

**Comment by puneeth2020** [comment]
OK... Merging branch77 now.

**Comment by puneeth2020** [comment]
Successfully merged.

**Pull request merged**